### ISSUE TITLE
Instructions for Chapter 1

### ISSUE THREAD

**Issue by khtan** [PacktPublishing/Hands-On-Microservices-with-CSharp-8-and-.NET-Core-3-Third-Edition#10]
Hi,
I am a newbie and would appreciate a little instruction for using the code in Chapter 01. Eg how to build, how to run, how to run tests ( if any ).
Thanks,

**Comment by khtan** [comment]
Or if the info already exists, where in the repository do I find it. 
Thanks,

**Comment by garora** [comment]
> Hi,
> I am a newbie and would appreciate a little instruction for using the code in Chapter 01. Eg how to build, how to run, how to run tests ( if any ).
> Thanks,

To run and execute the code, the following are the prerequisites:

- Visual Studio 2019

- Setting up .NET Core 3.0

To install and run these code-examples you need to install Visual Studio 2019 (preferred
IDE). To do so, follow these instructions:
1. Download Visual Studio 2019 (Community Edition (free)) from the download
link mentioned with installation instructions: [https://docs.microsoft.com enus/
visualstudio/install/install- visual- studio](https://docs.microsoft.com/enus/visualstudio/install/install-visual-studio)
2. Follow the installation instructions mentioned thereon.
3. Multiple versions are available for Visual Studio installation. We are using Visual
Studio for Windows.
Setting up .NET Core 3.0
If you do not have .NET Core 3.0 installed, you can download it from the link: [https://www.microsoft.com/net/download/windows](https://www.microsoft.com/net/download/windows)

**Comment by garora** [comment]
> Or if the info already exists, where in the repository do I find it.
> Thanks,

Every chapter has a dedicated section for this, you can check **Technical Requirements** section for all the required instructions. Hope it works for you.

**Comment by khtan** [comment]
Thank you for your replies. I am able report success and a possible misunderstanding.

**Success:**

In Chapter 1, I have to use VS Community 2019 to open the solution to build.
After building, I can run the website at localhost:38606. See picture attached.
![website-picture](https://user-images.githubusercontent.com/8535034/94613358-56690480-0259-11eb-9f9e-fff9f64fdf71.png)

**Misunderstanding:**
I also tried to build it using "dotnet build" but that failed. **In hindsight,** this is the monolith app, so it has not been ported to work for .net core and so I should not expect "dotnet build" to work.

   > dotnet build
   > Microsoft (R) Build Engine version 16.7.0+7fb82e5b2 for .NET
   > Copyright (C) Microsoft Corporation. All rights reserved.
   > 
   >   Determining projects to restore...
   >   Nothing to do. None of the projects specified contain packages to restore.
   > c:\cprojects\github\microservices\Hands-On-Microservices-with-CSharp-8-and-.NET-Core-3-Third-Edition\Chapter 01\FlixOne.BookStore\FlixOne.BookStore\FlixOne.BookStore.csproj(313,3): error MSB4019: The imported project "c:\Program Files\dotnet\sdk\3.1.402\Microsoft\VisualStudio\v16.0\WebApplications\Microsoft.WebApplication.targets" was not found. Confirm that the expression in the Import declaration "c:\Program Files\dotnet\sdk\3.1.402\Microsoft\VisualStudio\v16.0\WebApplications\Microsoft.WebApplication.targets" is correct, and that the file exists on disk.
   > 
   > Build FAILED.
   > 
   > c:\cprojects\github\microservices\Hands-On-Microservices-with-CSharp-8-and-.NET-Core-3-Third-Edition\Chapter 01\FlixOne.BookStore\FlixOne.BookStore\FlixOne.BookStore.csproj(313,3): error MSB4019: The imported project "c:\Program Files\dotnet\sdk\3.1.402\Microsoft\VisualStudio\v16.0\WebApplications\Microsoft.WebApplication.targets" was not found. Confirm that the expression in the Import declaration "c:\Program Files\dotnet\sdk\3.1.402\Microsoft\VisualStudio\v16.0\WebApplications\Microsoft.WebApplication.targets" is correct, and that the file exists on disk.
   >     0 Warning(s)
   >     1 Error(s)
   > 
   > Time Elapsed 00:00:00.44
   > 

In short, I am OK with Chapter 01.

**Comment by garora** [comment]
Glad to know that your issue is resolved.
